search.html

```diff
@@ -215,25 +216,35 @@
 
 <p>When OMERO adds data to it's search index, it processes the text and looks for keywords or “tokens” to index.</p> 
 
-<p>In general it does this by splitting text by any character that is not a letter or number. For example, an image name such as AuroraB-siRNAi_control_01.tiff will be split into 5 tokens: AuroraB, siRNAi, control, 01, tiff.</p>
+<p>In general it does this by splitting text by any character that is not a letter or number. For example, an image name such as <span class="code-example">AuroraB-siRNAi_control_01.tiff</span> will be split into 5 tokens:</p>
+
+<p class="code-example">AuroraB, siRNAi, control, 01, tiff.</p>
 
 <p>This image will be returned by a search for any of those tokens, but will be ranked higher in the search results if multiple tokens are matched.</p>
 
 <p>For example, searching for:</p>
 
-<p class="indent1">siRNAi AuroraB</p>
+<p class="indent1"><span class="code-example">siRNAi AuroraB</span></p>
+
+<p>will find any images that have tokens of:</p>
+<p><span class="code-example">siRNAi OR AuroraB</span></p>
+<p>but images that contain both these keywords will be returned ahead of other results.</p>
 
-<p>will find any images that have tokens of siRNAi OR AuroraB, but images that contain both these keywords will be returned ahead of other results. Searching for siRNAi AND AuroraB will only match images that have both tokens.</p>
+<p>Searching for:</p>
+<p><span class="code-example">siRNAi AND AuroraB</span></p>
+<p>will only match images that have both tokens.</p>
 
-<p>Search queries themselves are split into tokens as part of the search. Therefore, you expect to find an image named AuroraB-siRNAi_control_01.tiff with any of the following searches:</p>
+<p>Search queries themselves are split into tokens as part of the search. Therefore, you expect to find an image named <span class="code-example">AuroraB-siRNAi_control_01.tiff</span> with any of the following searches:</p>
 
 <ul class="bullet-list">
-<li>AuroraB_siRNAi</li>
-<li>AuroraB-siRNAi</li>
-<li>AuroraB siRNAi</li>
-<li>siRNAi AuroraB (Order is ignored for separate keywords)</li>
+<li class="code-example">AuroraB_siRNAi</li>
+<li class="code-example">AuroraB-siRNAi</li>
+<li class="code-example">AuroraB siRNAi</li>
+<li class="code-example">siRNAi AuroraB</li>
 </ul>
 
+<p>Order is ignored for separate keywords.</p>
```

Comment:
Are you sure that order is ignored for separate keywords? I thought that Lucene will tend to try and find results where the keyword matches are in the same order as the query? @joshmoore?